src/components/pages/document/create/resource-url/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { TextBoxField } from "@components/form/text";
+import useTranslation from "next-translate/useTranslation";
+import React from "react";
+
+
+
+export default function ExternalUrl() {
+    const { t } = useTranslation();
+
+    return (
+        <TextBoxField name="externalUrl" label={t("document:upload.url")} isRequired={true} />
+    );
+}
```

Comment:
I think this is used only at one place can you just put it there without extracting into seprate component?